client/client-contract-test/test/client-sdk.spec.ts

```diff
@@ -143,14 +142,20 @@ before((done) => {
 });
 
 describe("The Javascript SDK", () => {
-  const client = new OrbsClient(API_ENDPOINT, SENDER_ADDRESS, TIMEOUT);
-  const contract = new OrbsContract(client, CONTRACT_NAME);
+  const keyPairStub = stubObject<ED25519Key>(new ED25519Key(), ["sign"]);
```

Comment:
I think it would be better to initialize a real instance of `ED2551Key` with a private/public key because you'd have to do it in Java code anyway.